packages/block-editor/src/components/block-parent-selector/style.scss

```diff
@@ -0,0 +1,10 @@
+.block-editor-block-parent-selector {
+	background: $white;
+	border: 1px solid $dark-gray-primary;
+	border-radius: $radius-block-ui;
+
+	.block-editor-block-parent-selector__button {
```

Comment:
Is the nesting needed here?

Reply:
Yes, the specificity allows use to override the margin on the button. Without that, the button is misaligned.